timeseries/src/main/scala/com/criteo/cuttle/timeseries/TimeSeriesScheduler.scala

```diff
@@ -261,6 +261,16 @@ case class TimeSeriesScheduler(logger: Logger) extends Scheduler[TimeSeries] wit
   }
 
   def start(workflow: Workflow[TimeSeries], executor: Executor[TimeSeries], xa: XA, logger: Logger): Unit = {
+    logger.info("validate workflow before start")
+    TimeSeriesUtils.validate(workflow) match {
+      case Left(errors) =>
+        errors.foreach(logger.error(_))
+        logger.error("exit with error code 1")
+        System.exit(1)
```

Comment:
Why System.exit? Because you can start several projects in a singel JVM it's a bit strange to produce a global effect like that from inside the lib. Would be better the throw an exception (or return a Try) and let the launcher fail if it does not handle it.

Reply:
In that case we should handle the error in CuttleProject if the call to scheduler.start fails, and shutdown the server which is already started up. Currently if scheduler.start throws, the app boots normally.